client/src/components/FootyPoints.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Label } from 'semantic-ui-react';
+const logo = require('./logo.png');
+
+function FootyPoints (props) {
+    return (
+        <Label as='a' image>
```

Comment:
Why are we rending Label as an anchor tag?

Reply:
Bc my brain is small and I just copied + pasted the code from the example